src/Solvers/conjugate_gradient_poisson_solver.jl

```diff
@@ -1,4 +1,4 @@
-using Oceananigans.Operators: divᶜᶜᶜ, ∇²ᶜᶜᶜ 
+using Oceananigans.Operators
```

Comment:
Is there a reason to import the whole module here? I don't feel too strongly about it, but in general I think it makes the code clearer to be specific about what is being imported from each module.

Reply:
Well, we need quite a few metrics for the asymptotic preconditioner. This is actually a bug on main, we can't use the asymptotic preconditioner because of the missing imports

Reply:
Here:

https://github.com/CliMA/Oceananigans.jl/blob/3f62872b1dee461945c7afa9865619a9413ad473/src/Solvers/conjugate_gradient_poisson_solver.jl#L191-L196